README.md

```diff
@@ -1,82 +1,75 @@
-# My Awesome Step
+# Export iOS and tvOS Xcode archive
 
-My Awesome Step is a solid starting code base for
-a new Step.
+[![Step changelog](https://shields.io/github/v/release/bitrise-steplib/steps-export-xcarchive?include_prereleases&label=changelog&color=blueviolet)](https://github.com/bitrise-steplib/steps-export-xcarchive/releases)
 
+Export iOS and tvOS IPA from an existing Xcode archive
 
-## How to use this Step
+<details>
+<summary>Description</summary>
 
-Can be run directly with the [bitrise CLI](https://github.com/bitrise-io/bitrise),
-just `git clone` this repository, `cd` into it's folder in your Terminal/Command Line
-and call `bitrise run test`.
+Exports an IPA from an existing iOS and tvOS `.xcarchive` file.
+</details>
 
-*Check the `bitrise.yml` file for required inputs which have to be
-added to your `.bitrise.secrets.yml` file!*
+## 🧩 Get started
 
-Step by step:
+Add this step directly to your workflow in the [Bitrise Workflow Editor](https://devcenter.bitrise.io/steps-and-workflows/steps-and-workflows-index/).
 
-1. Open up your Terminal / Command Line
-2. `git clone` the repository
-3. `cd` into the directory of the step (the one you just `git clone`d)
-5. Create a `.bitrise.secrets.yml` file in the same directory of `bitrise.yml` - the `.bitrise.secrets.yml` is a git ignored file, you can store your secrets in
-6. Check the `bitrise.yml` file for any secret you should set in `.bitrise.secrets.yml`
-  * Best practice is to mark these options with something like `# define these in your .bitrise.secrets.yml`, in the `app:envs` section.
-7. Once you have all the required secret parameters in your `.bitrise.secrets.yml` you can just run this step with the [bitrise CLI](https://github.com/bitrise-io/bitrise): `bitrise run test`
+You can also run this step directly with [Bitrise CLI](https://github.com/bitrise-io/bitrise).
 
-An example `.bitrise.secrets.yml` file:
+### Example
 
-```
-envs:
-- A_SECRET_PARAM_ONE: the value for secret one
-- A_SECRET_PARAM_TWO: the value for secret two
-```
-
-## How to create your own step
+Archive, then export both development and app-store IPAs:
 
-1. Create a new git repository for your step (**don't fork** the *step template*, create a *new* repository)
-2. Copy the [step template](https://github.com/bitrise-steplib/step-template) files into your repository
-3. Fill the `step.sh` with your functionality
-4. Wire out your inputs to `step.yml` (`inputs` section)
-5. Fill out the other parts of the `step.yml` too
-6. Provide test values for the inputs in the `bitrise.yml`
-7. Run your step with `bitrise run test` - if it works, you're ready
+```yaml
+steps:
+- xcode-archive:
+    title: Archive and export development IPA
+    inputs:
+    - export_method: development
+- export-xcarchive:
+    title: Export app-store IPA
+    inputs:
+    - archive_path: $BITRISE_XCARCHIVE_PATH # this env var is an output of the previous step
```

Comment:
NOTE: might `output of the previous xcode-archive step` would be even more strathforward